ding_reservation.module

```diff
@@ -569,6 +569,7 @@ function ding_reservation_reservations_form($form, &$form_state, $reservations,
           '#id' => $reservation->id,
           '#title' => $title,
           '#cover' => field_view_field('ting_object', $entity, 'ting_cover', 'user_list'),
+          '#disabled' => $reservation->reservation_type == 'interlibrary_loans',
```

Comment:
Please change this to `'#disabled' => TRUE,`

The equality check is unneeded as `$reservation->reservation_type` is already checked in the switch statement and this is in the `DING_RESERVATION_INTERLIBRARY_LOANS` case. Comparing against a string value is also dangerous as the rest of the code uses predefined constants.